scripts/run_certbot

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# The script for renewing the SSL certificates
+
+certbot --nginx renew > /proc/1/fd/1
```

Comment:
```suggestion
exec certbot --nginx renew
```
Unless there is some important reason for this redirect?

as for exec - AFAIK otherwise the exit code will be lost and this script will always return 0

Reply:
I assumed that sending cron jobs's output to the container's stdout will be handy for monitoring certbot's activity. But now that you asked this, I started to question that decision, cause it will lead to mixed logs, which can be misleading. I will remove the redirect

Reply:
oh, I see, since this is executed by crond in the background

I guess it would make sense, then again stderr would be at minimum as important to redirect

can you redirect both then?

Reply:
Added. I started to hesitate cause I thought about some potential log collecting problems later on. I guess it can be easily handled though, cause nginx logs have different structure